RELEASE.md

```diff
@@ -1,3 +1,46 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* tf.data users can now use `tf.data.experimental.AUTOTUNE` for the num_parallel_calls argument of `tf.data.Dataset.interleave`, `tf.data.Dataset.map`, and `tf.data.experimental.map_and_batch` to dynamically tune the level of parallelism at runtime.
+* Keras models can now be directly exported to the SavedModel format(`tf.contrib.saved_model.save_keras_model()`) and used with Tensorflow Serving.
+* TensorFlow builds with XLA support by default, does not enable XLA by default, but makes it accessible via default binary builds.
+
+## Bug Fixes and Other Changes
+
+* tf.data:
+  * tf.data users can now represent, get, and set options of TensorFlow input pipelines using `tf.data.Options()`, `tf.data.Dataset.options()`, and `tf.data.Dataset.with_options()` respectively.
+  * All C++ code moves to the `tensorflow::data` namespace.
+  * Adding support for `num_parallel_calls` to `tf.data.Dataset.interleave`.
+* `tf.contrib`:
+  * Removed `tf.contrib.linalg`. `tf.linalg` should be used instead.
+  * Replace any calls to `tf.contrib.get_signature_def_by_key(metagraph_def, signature_def_key)` with `meta_graph_def.signature_def[signature_def_key]`. Catching a ValueError exception thrown by `tf.contrib.get_signature_def_by_key` should be replaced by catching a KeyError exception.
+* `tf.contrib.data`
+  * deprecated, and replaced by tf.data.experimental.
+* Keras:
+  * Keras model now supports evaluating with a `tf.data.Dataset`.
```

Comment:
Seems worth moving this into major features.

Reply:
Done.